src/build/env.py

```diff
@@ -187,6 +188,25 @@ def _create_isolated_env_virtualenv(path):  # type: (str) -> Tuple[str, str]
 # venv only exists on Python 3+
 if sys.version_info >= (3,):  # noqa: C901
 
+    once = functools.lru_cache(maxsize=None)
+
+    @once
+    def _fs_supports_symlink():  # type: () -> bool
+        """Return True if symlinks are supported"""
+        # Using definition used by venv.main()
+        if os.name != 'nt':
+            return True
+
+        # Windows may support symlinks (setting in Windows 10)
+        with tempfile.NamedTemporaryFile(prefix='TmP') as tmp_file:
```

Comment:
Use a prefix which points to build.

Reply:
I just inherited [this code](https://github.com/pypa/build/commit/dad5f626742b5643ae0d01e386855cdb28f34aa4#diff-4f1bb46d65de7fbb1573dfc2a2bca9f63c7b03d55e74c53339f0c8ee4fa054d8R21), happy to change.